source/_posts/2014-08-08-CocoaPods-Caching.markdown

```diff
@@ -1,5 +1,5 @@
 ---
-layout: full_width_post
```

Comment:
in the case of the three posts being edited here, there is no longer a 'full_width_post' layout so I've just edited them to be normal 'post' layouts instead.